packages/grafana-ui/src/components/uPlot/MicroPlot.scss

```diff
@@ -0,0 +1,241 @@
+// NOTE:
+// This file is copied from:
+// https://github.com/leeoniya/uPlot/blob/master/src/uPlot.css
+// and then run through an online converter
```

Comment:
why not injest the styles directly? this seems like unneccessary upkeep.

Reply:
we have a light and dark theme (dark is most common) -- I would love a solution that could simply inject, but this was my quick hack.

Any better ideas?

Reply:
my initial thought is for me to author the css in a nested format and compile it using goober+extractCss [1]. i never really liked that the final css lives in `src` rather than `dist`. so that kills 2 birds. as for theming, `:root` with css variables might be a good native option. another option is to also offer a dark theme, but that might be a slippery slope (depends on how many vars end up in the css). i'd like to stay away from more complex/heavy devdeps such as sass or less.

on the js side, you can provide all the default colors at chart init time, but in-place stroke/fill toggling (without a chart rebuild) would require more support from uPlot - i'm not sure if this is worth supporting.

let's discuss it further in https://github.com/leeoniya/uPlot/issues/113.

[1] https://github.com.cnpmjs.org/cristianbote/goober#extractcsstarget